README.rst

```diff
@@ -20,8 +20,22 @@ Electrum - Lightweight Bitcoin client
 Getting started
 ===============
 
-Electrum is a pure python application. If you want to use the
-Qt interface, install the Qt dependencies::
+Electrum is a pure python application. Electrum is available for
+`Ubuntu and other snap supported Linux distributions
+<https://snapcraft.io/docs/core/install>`_ in the snap store and
+can be simply installed with:
```

Comment:
I think you need to end the line with :: in order to make the next line a code block.